### PR TITLE
update strings translations for flowchart

### PR DESCRIPTION
**Pull request by Avuxo**
these haven't been TLC'd yet

i'm gonna copy my notes from the spreadsheet over into comments

### CODE REVIEW

File: system_strings/sysmes_text.en [hunk 1]
```diff
@@ -814,77 +814,77 @@ I Kiss the Vampire Princess
...
+Lost World
+First, I need to hide.
+Logic Error
+The ‘Point’ of the World
+Homecoming Gravitational Field
+Has nothing to do with me at all
+Love Letter
+Waking
+Mystic eyes, wide awake
+Inverse Channel/Celestial Birth
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:881]: Getting this to fit was a bitch I really like Celestial Birth but don't like inverse channel

File: system_strings/sysmes_text.en [hunk 2]
```diff
@@ -814,77 +814,77 @@ I Kiss the Vampire Princess
...
+Garden of the Giants
+Struggle of the ants
+Don’t give up.
+No choice but to escape.
+Lost World
+First, I need to hide.
+Logic Error
+The ‘Point’ of the World
+Homecoming Gravitational Field
+Has nothing to do with me at all
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:877]: same text as 40604 but too long

File: system_strings/sysmes_text.en [hunk 3]
```diff
@@ -814,77 +814,77 @@ I Kiss the Vampire Princess
...
+Sniping
+Impact
+The voice of Roa
+180 second decisive barrier fight
+Krake III
+Snow of chalk
+Manifestation
+Final warning
+So what? Keep your mouth shut.
+Fine, then what should I do?
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:866]: same as 42761

File: system_strings/sysmes_text.en [hunk 4]
```diff
@@ -814,77 +814,77 @@ I Kiss the Vampire Princess
...
+Cries of the planet.
+Sniping
+Impact
+The voice of Roa
+180 second decisive barrier fight
+Krake III
+Snow of chalk
+Manifestation
+Final warning
+So what? Keep your mouth shut.
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:865]: same text as 42760 but too long

File: system_strings/sysmes_text.en [hunk 5]
```diff
@@ -814,77 +814,77 @@ I Kiss the Vampire Princess
...
+Death into the world.
+Farewell
+The remains of the dream
+I can’t come to dislike it.
+Event Storage
+Cries of the planet.
+Sniping
+Impact
+The voice of Roa
+180 second decisive barrier fight
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:860]: sounds like complete ass and makes 0 sense, but i didnt find a great way to fit that much information in so few characters

File: system_strings/sysmes_text.en [hunk 6]
```diff
@@ -814,77 +814,77 @@ I Kiss the Vampire Princess
...
-じゃあ、どうしろって言うんだ。
+Don’t accompany Arcueid.
+Fallen True Ancestor
+If tomorrow comes.
+Death into the world.
+Farewell
+The remains of the dream
+I can’t come to dislike it.
+Event Storage
+Cries of the planet.
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:856]: this is what the TL says in ff7, if "screams" or w/e is better, feel free to change

File: system_strings/sysmes_text.en [hunk 7]
```diff
@@ -814,77 +814,77 @@ I Kiss the Vampire Princess
...
-それがどうした、黙ってろ。
-じゃあ、どうしろって言うんだ。
+Don’t accompany Arcueid.
+Fallen True Ancestor
+If tomorrow comes.
+Death into the world.
+Farewell
+The remains of the dream
+I can’t come to dislike it.
+Event Storage
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:855]: this is used a ton in day 14 but that hasnt been TLC'd yet. make consistent with term

File: system_strings/sysmes_text.en [hunk 8]
```diff
@@ -814,77 +814,77 @@ I Kiss the Vampire Princess
...
+Before escaping
+Judge in the other depths
+Escape forward.
+Escape back.
+Quiet ocean
+Remove my glasses.
+Don’t remove my glasses.
+Avenger
+The French Incident
+Homecoming from the rose garden
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:831]: I have _no_ idea how to make 生還 fit

File: system_strings/sysmes_text.en [hunk 9]
```diff
@@ -776,36 +776,36 @@ The Last Supper
...
-壊れた椅子
-止まった時計
-夜歩く
-カルマの清算
+11th day ― Morning
+A familiar classroom
+A broken chair
+A stopped clock
+Night walk
+Balancing of karma
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:784]: 清算 is like cancelling out debt through liquidation in finance.

File: system_strings/sysmes_text.en [hunk 10]
```diff
@@ -689,30 +689,30 @@ Death.
...
+A promise is a promise.
+Respond clearly.
+Respond calmly.
+Cooly respond.
+Head on confrontation
+Melty blood
+Victory is certain.
+Save Ciel-senpai
+I won't like you anymore!
+Don't say the wrong thing
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:713]: jp is from 42725 but didn't even remotely fit

File: system_strings/sysmes_text.en [hunk 11]
```diff
@@ -689,30 +689,30 @@ Death.
...
+Indecisivieness
+A promise is a promise.
+Respond clearly.
+Respond calmly.
+Cooly respond.
+Head on confrontation
+Melty blood
+Victory is certain.
+Save Ciel-senpai
+I won't like you anymore!
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:712]: text is from 23924

File: system_strings/sysmes_text.en [hunk 12]
```diff
@@ -689,30 +689,30 @@ Death.
...
+9th day ― Morning
+Morning greetings.
+The wet infirmary.
+Painless and effective
+Painful and ineffective
+The promised time has come.
+discommunication
+A receieved text
+I'll go see Noel-sensei
+I'll do as she did.
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:701]: text is from 23924

File: system_strings/sysmes_text.en [hunk 13]
```diff
@@ -657,21 +657,21 @@ Encounter
...
+Ciel and Noel
+Senpai leaves the infirmary.
+Senpai's suspicious actions.
+I'm curious, I'll follow Senpai.
+Let's head back, Senpai.
+Ciel-san in India.
+I won't lie.
+I just can't leave Arcueid alone.
+That person is my dear Senpai.
+Prayers to the lord
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:673]: "主は来ませり" is the japanese translation for "The lord has come" in Joy to the World, this tl is kinda dry and kinda incorrect but i dont know what else fits. "The lord has heard your prayer" maybe? The unusual ~ませり is also used for a later title we've translated as being a line from Dante so maybe this should get a similar treatment?

File: system_strings/sysmes_text.en [hunk 14]
```diff
@@ -657,21 +657,21 @@ Encounter
...
+8th day - Morning
+What's the plan from here?
+The bane of my existence
+Shallow, fleeting dream
+Ciel and Noel
+Senpai leaves the infirmary.
+Senpai's suspicious actions.
+I'm curious, I'll follow Senpai.
+Let's head back, Senpai.
+Ciel-san in India.
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:669]: This is the scene in the curry shop. Might just need to be something else entirely.

File: system_strings/sysmes_text.en [hunk 15]
```diff
@@ -657,21 +657,21 @@ Encounter
...
-戻りましょう、先輩。
-シエルさん、インドですよ。
-嘘は言ってない。
-アルクェイドの方がほっとけない。
-あの人は、大切な先輩だ。
-主に祈りは届きませり
-1989年
+8th day - Morning
+What's the plan from here?
+The bane of my existence
```
Comment (Avuxo) [inline system_strings/sysmes_text.en:662]: "my" because char limits

**Review by rschlaikjer — COMMENTED** [review]
otsu

File: system_strings/sysmes_text.en [hunk 16]
```diff
@@ -776,36 +776,36 @@ The Last Supper
...
+You're even skipping school?
+How do you compartmentalize?
+You control the student council?
+Everybody's Ciel-senpai
+It's like a dream
+Returning to my room.
+Kohaku-san is in the way.
+Kohaku-san is right.
+Kohaku-san is right.
+Unceasing headache.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply system_strings/sysmes_text.en:806]: when i was writing this all i had was "ceaseless discharge" from dark souls in my head

**Pull request merged**